examples/example_2.mff/subjects/luyibin.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
Okay - yeah, this file needs to be scrubbed. We cannot have identifiable information in the repository. Period. Full stop.

Please replace anything identifiable in this file with anonymized data, and I'll work with the team to clean up the repository.